app/scripts/modules/core/src/managed/Environments2.tsx

```diff
@@ -56,7 +56,7 @@ export const UISwitcher = () => {
 };
 
 const tabsInternal: { [key in Routes]: string } = {
-  overview: 'Overview',
+  overview: 'Test',
```

Comment:
Is it temporary? :)

Reply:
Ah no :( Thanks for catching it. I was testing local development and forgot to remove it.